cns/restserver/restserver_test.go

```diff
@@ -689,3 +689,23 @@ func TestGetInterfaceForNetworkContainer(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetNumOfCPUCores(t *testing.T) {
+	fmt.Println("Test: getNumberOfCPUCores")
+
+	req, err := http.NewRequest(http.MethodGet, cns.NumberOfCPUCoresPath, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	w := httptest.NewRecorder()
+	mux.ServeHTTP(w, req)
+	var numOfCoresResponse cns.NumOfCPUCoresResponse
```

Comment:
Declare vars upfront?
var (
)

Reply:
Can you respond to the review comment rather than marking it resolved directly if you are not going to act on it ?